src/components/Dialogs/MetricInfoDialog.stories.tsx

```diff
@@ -0,0 +1,56 @@
+import React, { useEffect, useState } from 'react';
+import { DialogMain, MetricInfoDialogInner } from 'components/Dialogs';
+import regions from 'common/regions';
+import { getSourcesForMetric } from 'common/utils/provenance';
+import { Projection } from 'common/models/Projection';
+import { getProjectionForRegion } from 'components/NewLocationPage/SparkLineBlock/utils';
+import { Metric } from 'common/metricEnum';
+import { getMetricModalContent } from './utils';
+import { useDialog } from 'common/hooks';
+
+export default {
+  title: 'Below the fold/Metric info modal',
+  component: DialogMain,
+};
+
+export const Example = () => {
+  const region = regions.findByFipsCodeStrict('25');
+
+  const [projection, setProjection] = useState<Projection>();
+
+  useEffect(() => {
+    const fetchProjection = () => getProjectionForRegion(region);
+    fetchProjection().then(setProjection);
+  }, [region]);
```

Comment:
we can hardcode the projection since the region is not dynamic right?

```tsx
const region = regions.findByFipsCodeStrict('25');
const projection = getProjectionForRegion(region);

// ...

const provenance = getSourcesForMetric(projection.annotations, exampleMetric);
// ...
```

Reply:
`getProjectionForRegion` is async

Reply:
either way, fetching projections/this `useEffect` etc are only for the sake of wiring this up in storybook and grabbing the region's provenance info. when this is implemented in ChartBlock, projections will already be available

Reply:
[this is where provenance is currently grabbed](https://github.com/covid-projections/covid-projections/blob/develop/src/components/LocationPage/ChartBlock.tsx#L34)